src/go/pt-k8s-debug-collector/dumper/dumper.go

```diff
@@ -451,7 +451,7 @@ func (d *Dumper) getPodSummary(resource, podName, crName string, namespace strin
 		}
 		ports = port + ":27017"
 		summCmdName = "pt-mongodb-summary"
-		summCmdArgs = []string{"--username='" + user + "'", "--password='" + pass + "'", "--authenticationDatabase=admin", "127.0.0.1:" + port}
+		summCmdArgs = []string{"--username=" + user, "--password=" + string(pass), "--authenticationDatabase=admin", "127.0.0.1:" + port}
```

Comment:
this probably fail if password contains `"`. for all possible symbols please check: https://github.com/percona/percona-xtradb-cluster-operator/blob/main/pkg/controller/pxc/secrets.go#L98-L102

Reply:
This is MongoDB but everything works for password '8^ye<*2*cLsQ%f]"d<K '. Tested with MongoDB, PXC, and MySQL.